### PR TITLE
Add subtis extension

### PR DESCRIPTION
**Pull request by lndgalante**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Could you add the missing `package-lock.json` file

**Comment by lndgalante** [comment]
Hey @pernielsentikaer sure!

**Comment by lndgalante** [comment]
@pernielsentikaer `package-lock.json` added, thank you!

**Comment by pernielsentikaer** [comment]
Do you need help fixing the [CI error](https://github.com/raycast/extensions/actions/runs/11539046029/job/32118253479?pr=14953)?

**Comment by lndgalante** [comment]
@pernielsentikaer CI check is passing!

File: extensions/subtis/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "subtis",
+  "title": "Subtis",
+  "description": "El subtítulo que buscabas",
```
Comment (pernielsentikaer) [inline extensions/subtis/package.json:5]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.

File: extensions/subtis/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "author": "leonardo_galante",
+  "license": "MIT",
+  "icon": "command-icon.png",
+  "version": "0.4.3",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Buscar Subtitulo",
+      "description": "Encontra el subtítulo que buscas",
```
Comment (pernielsentikaer) [inline extensions/subtis/package.json:14]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.

File: extensions/subtis/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+    }))
+    .with(500, () => ({
+      title: "Error Inesperado",
+      description: "Estamos haciendo arreglos del servicio",
+    }))
+    .otherwise(() => ({
+      title: "Error desconocido",
+      description: "Estamos haciendo arreglos del servicio",
+    }));
+}
```
Comment (pernielsentikaer) [inline extensions/subtis/src/index.tsx:63]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.

**Pull request closed without merging**